python/hidet/runtime/compiled_graph.py

```diff
@@ -208,6 +208,10 @@ def _create_outputs(self, inputs):
 
         outputs = []
         for output_index, (exec_idx, sig) in enumerate(zip(self.graph_execution.outputs_index, self.meta.outputs)):
+            if sig.alias is not None:
+                outputs.append(outputs[sig.alias])
+                continue
+
```

Comment:
We can do the checking alternatively like
```
            if exec_idx in self.graph_execution.inputs_index:
                outputs.append(inputs[self.graph_execution.inputs_index.index(exec_idx)])
            elif exec_idx in self.graph_execution.weights_index:
                outputs.append(self.weights[self.graph_execution.weights_index.index(exec_idx)])
            elif exec_idx != self.graph_execution.outputs_index.index(exec_idx): # <--------
                outputs.append(outputs[self.graph_execution.outputs_index.index(exec_idx)])
            else:
```
so we do not need to modify the graph building process or an alias attribute.

How do you think @Aalanli ?

Reply:
The suggested change doesn't work, but I see how to fix it. 
Yeah, I think this way is better as well.